catalog/java/io/material/catalog/bottomappbar/res/layout/cat_bottomappbar_fragment.xml

```diff
@@ -42,19 +42,22 @@
       app:layout_anchor="@id/bar"
       app:srcCompat="@drawable/ic_add_24px"/>
 
-  <FrameLayout
+  <RelativeLayout
```

Comment:
Use LinearLayout instead? RelativeLayout is usually less favorite because of the performance concern.